cody.el

```diff
@@ -862,30 +958,42 @@ and the start of the overlay."
          (spot (point))
          (trigger-kind (if (called-interactively-p 'interactive)
                            "Invoke" "Automatic")))
```

Comment:
minor: Would this be more elispy if "Invoke" and "Automatic" where symbols? Or is that a pain to then marshal as the strings expected by the agent?

Reply:
I agree that they are a bit of an eyesore, but I tried it, and apparently the logic for json marshalling is way down in the C layer, and doesn't support symbols for value types (only keys).